core-lib/TestSuite/Serialization.ns

```diff
@@ -258,5 +258,5 @@ class SerializationTests usingPlatform: platform testFramework: minitest = Value
       (assert: ppc promise resolvedWith: 42).
   )
 
-  ) : ( TEST_CONTEXT = () )
+  ) : ( DEACTIVATED_TEST_CONTEXT = () )
```

Comment:
Please leave a TODO comment here to explain this.

And similar to the targets, do we really need to deactivate it here to get the desired effect?
I'd think changes to the CI setup are sufficient, no? (and perhaps in `SomTests`)